intera_interface/src/intera_interface/limb.py

```diff
@@ -146,10 +174,14 @@ def __init__(self, limb="right", synchronous_pub=False):
                    "from %s") % (self.name.capitalize(), ns + 'endpoint_state')
         intera_dataflow.wait_for(lambda: len(self._cartesian_pose.keys()) > 0,
                                  timeout_msg=err_msg)
+        err_msg = ("%s limb init failed to get current tip_states "
+                   "from %s") % (self.name.capitalize(), ns + 'tip_states')
+        intera_dataflow.wait_for(lambda: self._tip_states is not None,
```

Comment:
- [ ] similar to the FKService dependency, you probably want to throw a note over to make sure to publish tip-states in sim